drivers/src/ecc384.rs

```diff
@@ -280,6 +280,39 @@ impl Ecc384 {
         Ok(pub_key)
     }
 
+    pub fn pcr_sign_flow(
+        &mut self,
+        data: &Ecc384Scalar,
+        trng: &mut Trng,
+    ) -> CaliptraResult<Ecc384Signature> {
+        let ecc = self.ecc.regs_mut();
+
+        // Wait for hardware ready
+        wait::until(|| ecc.status().read().ready());
+
+        // Copy digest
+        KvAccess::copy_from_arr(data, ecc.msg())?;
```

Comment:
I would expect that `pcr_sign` computes the message itself rather than taking as input. If you remove this, is the data that's output the same?